11ty/definitions/bottom-up.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Bottom-up
```

Comment:
I really appreciate [Harold's](https://github.com/HaroldDavis3) take on Bottoms up governance👉🏾 look [here](https://github.com/Underground-Railroad/magnificentMammals/blob/main/magnificentMammals/harold.md) I have compiled some of the writings I have found and enjoyed.